.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: Ruby
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Ruby ${{ matrix.ruby }}, Gemfile ${{ matrix.gemfile }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby:
+          - 2.7
+          - 2.6
+        gemfile:
+          - gemfiles/Gemfile.rails-5.0.x
+          - gemfiles/Gemfile.rails-5.1.x
+    env:
+      SQS_QUEUE_URL: 'sqlite3://tmp/sqewer.sqlite3'
+      AWS_REGION: 'eu-central-1'
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: install gems
+      run: |
+        gem install sqlite3
+        gem install bundler
+        bundle install --gemfile ${{ matrix.gemfile }}
+    - name: Tests
+      run: |
+        bundle exec rspec --tag "~sqs"
```

Comment:
Are we planning on running SQS tests any time soon?

Reply:
well, I don't know how can we do it, because it requires AWS credentials for this.

Maybe we can add credentials on Github secrets?

Reply:
(I will merge this PR, we can improve it later)